### PR TITLE
RuntimeClassName - Update DropDisabled[Alpha]Fields behaviour

### PR DESCRIPTION
**Pull request by sbezverk**
Signed-off-by: Serguei Bezverkhi <sbezverk@cisco.com>

See for reference issue: #72169

```release-note
None
```

/sig api-machinery
/kind cleanup
/release-notes-none

### CODE REVIEW

**Comment by sbezverk** [comment]
@liggitt Appreciate a lot if you could clarify what you mean.

File: pkg/api/pod/util.go [hunk 1]
```diff
@@ -397,3 +395,15 @@ func subpathInUse(podSpec *api.PodSpec) bool {
 	}
 	return false
 }
+
+// podPriorityInUse check if either of podSpec.Priority or podSpec.PriorityClassName
```
Comment (liggitt) [inline pkg/api/pod/util.go]: fix doc

**Comment by liggitt** [comment]
nit on godoc, lgtm otherwise

**Comment by sbezverk** [comment]
@liggitt Doc fixed, PTAL

**Pull request merged**